t/app/controller/report_as_other.t

```diff
@@ -150,19 +152,20 @@ subtest "Body user, has permission to add report as another (existing) user with
     push @users, $report->user;
 };
 
-subtest "Body user, has permission to add report as anonymous user" => sub {
+subtest "Superuser, can add report as anonymous user" => sub {
```

Comment:
Does this not mean we are losing the test coverage for body users reporting as anonymous users?

Reply:
Sort of, explicitly, but I figured that the functionality was covered by this test in combination with others.